### DISCUSSION TITLE
Possible to disable cache permanenty? (like a default -k switch)

### DISCUSSION THREAD

**Discussion by elisimpson** [ScoopInstaller/Scoop#5762] [Help and Questions]
I use the "-k" switch on every install and update command and I'm wondering if it would be possible to add something to scoop config to disable cache everywhere so I don't have to add that switch every time.

Perhaps something like `scoop config disable_cache $true`

**Comment by rashil2000** [comment]
You can create an alias:
```
scoop alias add kinstall 'scoop install $args -k'
```

and then use it like 

```
scoop kinstall <package-name>
```
